OSDOCS#17844: Document replacing a healthy etcd member (#112025)
openshift/openshift-docs: modules/replace-healthy-etcd-about.adoc

// Module included in the following assemblies:
//
// * backup_and_restore/control_plane_backup_and_restore/replacing-healthy-etcd-member.adoc
// * etcd/etcd-backup-restore/replace-healthy-etcd-member.adoc

:_mod-docs-content-type: CONCEPT
[id="replace-healthy-etcd-about_{context}"]
= About replacing a healthy etcd member

[role="_abstract"]
To replace a control plane node without disrupting etcd, remove a healthy member and add a replacement while the cluster remains operational. The procedure you follow depends on how your cluster was installed and whether it uses the Machine API and a control plane machine set.

[NOTE]
====

> If the etcd member is unhealthy because the machine is not running, the node is not ready, or the etcd pod is crashlooping, see "Replacing an unhealthy etcd member" in Additional resources instead of this procedure.

Reviewer: Per the OCP docs guidelines, we're not supposed to mention "the Additional resources section" or variants of that phrase. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-non-xref-situations

```suggestion
If the etcd member is unhealthy because the machine is not running, the node is not ready, or the etcd pod is crashlooping, see "Replacing an unhealthy etcd member".
```

> If you have lost the majority of your control plane hosts, see "Restoring to a previous cluster state" in Additional resources instead of this procedure.

Reviewer: ```suggestion
If you have lost the majority of your control plane hosts, see "Restoring to an earlier cluster state".
```
Removing reference to "Additional resources" and changing title to match the title of the procedure.
